internal/Picker/tests/Picker-specs.js

```diff
@@ -138,6 +162,51 @@ describe('Picker Specs', () => {
 		expect(actual).toBe(expected);
 	});
 
+	test('should register wheel event when \'joined\' if \'value\' is between \'max\' and \'min\'', () => {
+		const handleWheelEvent = jest.fn();
+		render(
+			<Picker index={0} joined max={3} min={0} noAnimation step={1} value={1} onWheel={handleWheelEvent} />
+		);
+		const picker = screen.getByLabelText('1 press ok button to change the value');
+
+		act(() => {
+			picker.focus();
+		});
+		fireEvent.wheel(picker, {deltaY: 10});
+
+		expect(handleWheelEvent).toHaveBeenCalled();
+	});
+
+	test('should register wheel event when \'joined\' even if \'value\' is \'max\' and \'deltaY\' is positive', () => {
+		const handleWheelEvent = jest.fn();
+		render(
+			<Picker index={0} joined max={3} min={0} noAnimation step={1} value={3} onWheel={handleWheelEvent} />
+		);
+		const picker = screen.getByLabelText('3 press ok button to change the value');
+
+		act(() => {
+			picker.focus();
+		});
+		fireEvent.wheel(picker, {deltaY: 10});
+
+		expect(handleWheelEvent).toHaveBeenCalled();
+	});
+
+	test('should register wheel event when \'joined\' even if \'value\' is \'min\' and \'deltaY\' is negative', () => {
```

Comment:
```suggestion
	test('should call onWheel event when \'joined\' and \'value\' is \'min\' and \'deltaY\' is negative', () => {
```
Also please fix this line.